config.json

```diff
@@ -1419,15 +1419,52 @@
         "nugetId": "Xamarin.Google.Dagger",
         "dependencyOnly": true
       },
+      {
+        "groupId": "com.google.code.findbugs",
+        "artifactId": "jsr305",
+        "version": "3.0.1",
+        "nugetVersion": "3.0.1",
```

Comment:
The most recent version on NuGet is `3.0.2`.

Reply:
completely new artifact in `config.json` so I messed something up.

fixed